packages/grpc-js/src/server-call.ts

```diff
@@ -16,38 +16,89 @@
  */
 
 import {EventEmitter} from 'events';
+import * as http2 from 'http2';
 import {Duplex, Readable, Writable} from 'stream';
+
 import {ServiceError} from './call';
+import {Status} from './constants';
 import {Deserialize, Serialize} from './make-client';
 import {Metadata} from './metadata';
 
+function noop(): void {}
+
+export type PartialServiceError = Partial<ServiceError>;
+
+type DeadlineUnitIndexSignature = {
+  [name: string]: number
+};
+
+const kGrpcMessageHeader = 'grpc-message';
+const kGrpcStatusHeader = 'grpc-status';
+const kGrpcTimeoutHeader = 'grpc-timeout';
+const kDeadlineRegex = /(\d{1,8})\s*([HMSmun])/;
```

Comment:
These `k`-prefixed constants don't match the coding style of the rest of the codebase. You should instead use `ALL_CAPS_SNAKE_CASE`.